src/interfaces/swig/SGBase.i

```diff
@@ -447,6 +472,25 @@ namespace shogun
 }
 
 %pythoncode %{
+def _internal_get_param(self, name):
+
+	for f in (self._get,
+			self._get_real,
+			self._get_int,
+			self._get_real_matrix,
+			self._get_real_vector,
+			self._get_int_vector
+			):
+		try:
+			return f(name)
+		except (SystemError, RuntimeError):
```

Comment:
why two errors? What is the context of each?

Reply:
self._get returns a SystemError and the other getters return a RuntimeError

Reply:
self._get returns a SystemError and the other getters return a RuntimeError

Reply:
You know why that is?

Reply:
And also the question: is that good? Maybe we should change that?

Reply:
I will have a better look during the week, but it seems that there are multiple definitions of how exceptions are handled in `swig_typemaps.i`. I will see if I can simplify this!

Reply:
ok, but seperate PR